exercises/minesweeper/src/lib.rs

```diff
@@ -1 +1,3 @@
-
+pub fn annotate(minefield: &[&str]) -> Vec<String> {
+    unimplemented!("\nAnnotate the given minefield with the number of surrounding mines in each square (blank if there are no surrounding mines):\n{:#?}\n", minefield);
```

Comment:
I was pondering this sentence. If someone read the phrase "surrounding mines in each square" as one unit (rather than the way the sentence should actually be read), it may not be the best.

I'll present another possibility: "Annotate each square of the given minefield with the number of surrounding mines"

Is it better, worse, makes no difference? I may need some input here because English is not my first language

Reply:
I think your wording is better.
Perhaps only a small change: "Annotate each square of the given minefield with the number of  mines that surround said square"

Reply:
That's a great change!